modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -29,20 +29,21 @@ endif::[]
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Connection Timeout a| Number |  |  +++50000+++ | 
-| Socket Timeout a| Number |  |  +++50000+++ | 
-| Access Key a| String |  |  | x
-| Secret Key a| String |  |  | x
-| Try Default AWSCredentials Provider Chain a| Boolean |  |  +++false+++ | 
-| Max Connections a| Number |  |  | 
-| Region Endpoint a| String |  |  +++US_EAST_1+++ | 
+| S3 Compatible Storage URL a| String |  +++The URL to connect to in case an S3 compatible storage is used. If blank, the default AWS S3 one will be used.+++ |  | 
```

Comment:
The URL to connect to when S3-compatible storage is used. If blank, the connector uses the default AWS S3 URL.

Reply:
Changed